libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.html

```diff
@@ -8,7 +8,7 @@
 
       <sky-data-view [viewId]="viewConfig.id">
         <sky-repeater expandMode="none">
-          @for (column of displayedColumnData; track column.id) {
+          @for (column of displayedColumnData; track column) {
```

Comment:
why are we tracking only the item and not a value inside the item?

Reply:
These are a fix for earlier PRs. The `track` tells Angular what to watch for to redraw the `for` loop. I misunderstood what it was doing and was viewing it as a unique identifier for the object - but the previous implementation would have meant that the repeater items would only update if a new object was added, something was removed, or an object's `id` property changed. This would have caused issues for things where mutable properties inside the object changed. 

This fix just reverts us back to the behavior prior to the control flow - where Angular watches the entire object.

Reply:
Agreed, Trevor. Good change.

Reply:
Some of what I said here isn't true after more research. However, I met with Sandhya and chatted with Steve and we decided to go ahead with these reverts to be safe on our conversions as the performance gain seems negligible based on local testing.